lib/capybara/screenshot/diff/test_methods.rb

```diff
@@ -72,19 +72,21 @@ def screenshot(name, skip_stack_frames: 0, **options)
           screenshot_full_name = build_full_name(name)
           job = build_screenshot_matches_job(screenshot_full_name, options)
 
-          return false unless job
+          unless job
+            if Screenshot::Diff.fail_if_new
+              raise_error("No existing screenshot found for #{screenshot_full_name}!\nTo stop seeing this error disable by Capybara::Screenshot::Diff.fail_if_new=false", caller(2))
```

Comment:
The second part of this message should be removed since is represents the default.  The error will only occur if the developer has activated this option.